### DISCUSSION TITLE
Faster telemetry recovery

### DISCUSSION THREAD

**Discussion by Hrisfpv** [ExpressLRS/ExpressLRS#1366] [Ideas]
I have noticed that a lot of the other rc protocols have instant telemetry recovery while on elrs at least on my setup runinng the r9m and 2.1.0 i have to wait at least 3-5 seconds to get telemetry.Small thing but i was wondering if it was possible to implement.

**Comment by schugabe** [comment]
Can you post more details about your setup and settings and how you notice the 3-5 seconds before you get telemetry?

**Reply by Hrisfpv** [reply]
Yup.Im running elrs 2.1 on an r9m 2019 and variety of frsky recievers.After plugging in the lipo on my quad i have to wait like 4-5 secconds until my radio says telemetry recovered.200hz and tried every telem ratio but still the same thing.Tried flashing one of my recievers using stlink but still no go.
